quantum/dynamic_keymap.c

```diff
@@ -35,7 +35,11 @@
 // Explicitly override it if the keyboard uses a microcontroller with
 // more EEPROM *and* it makes sense to increase it.
 #ifndef DYNAMIC_KEYMAP_EEPROM_MAX_ADDR
-#    define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR 1023
+#    ifdef __AVR_AT90USB1286__
```

Comment:
```suggestion
#    if defined(__AVR_AT90USB646__) || defined(__AVR_AT90USB647__) | defined(__AVR_AT90USB1286__) || defined(__AVR_AT90USB1287__)
```

Reply:
Seeing as these are the only supported AVRs with more than 1kB EEPROM (and there are unlikely to be more in future), this could probably do.